packages/admin/stubs/TableWidget.stub

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace {{ namespace }};
+
+use Closure;
+use Filament\Tables;
+use Filament\Widgets\TableWidget as BaseWidget;
+use Illuminate\Database\Eloquent\Builder;
+
+class {{ class }} extends BaseWidget
+{
+    protected function getTableHeading(): string | Closure | null
+    {
+        return '{{ class }}';
```

Comment:
I think that Ryan meant that the default heading should be generated by `Str::title()` instead of being copied from the class name - like we do on Page and Resource classes.

Reply:
Gotcha, that's been added.